src/libAtomVM/opcodesswitch.h

```diff
@@ -244,7 +249,7 @@ typedef dreg_t dreg_gc_safe_t;
         case COMPACT_XREG:                                                                          \
         case COMPACT_YREG:                                                                          \
             (dreg).index = first_byte >> 4;                                                         \
--            break;                                                                                 \
+             break;                                                                                 \
```

Comment:
I'm nor sure what's happening here on the diff.

Reply:
I also noticed that when reviewing it after the rebase.
We apparently missed it in 776 :(

https://github.com/atomvm/AtomVM/blame/8308b43fd0ba5cee396d46a243e76f71bdf2655e/src/libAtomVM/opcodesswitch.h#L245